src/auth/GetWindowsCert.ts

```diff
@@ -0,0 +1,32 @@
+import * as ca from "win-ca";
+import * as caApi from "win-ca/api";
+import * as crypto from "crypto";
+
+export const UseWindowsCerts = (id: string): string => {
```

Comment:
Since this is just one method, let's integrate it in the existing `Auth` class.

Reply:
Will work on this in the next update

Reply:
Integrated into Main Auth class